flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/plan/nodes/exec/stream/OverWindowTestPrograms.java

```diff
@@ -227,141 +227,63 @@ public class OverWindowTestPrograms {
-                                                            4L),
-                                                    Row.ofKind(
-                                                            RowKind.UPDATE_BEFORE,
-                                                            "key1",
-                                                            4L,
-                                                            400L,
-                                                            5L),
-                                                    Row.ofKind(
-                                                            RowKind.UPDATE_AFTER,
-                                                            "key1",
-                                                            4L,
-                                                            400L,
-                                                            8L),
-                                                    Row.ofKind(
-                                                            RowKind.UPDATE_BEFORE,
-                                                            "key1",
-                                                            5L,
-                                                            500L,
-                                                            10L),
-                                                    Row.ofKind(
-                                                            RowKind.UPDATE_AFTER,
-                                                            "key1",
-                                                            5L,
-                                                            500L,
-                                                            13L),
-                                                    Row.ofKind(
-                                                            RowKind.UPDATE_BEFORE,
-                                                            "key1",
-                                                            6L,
-                                                            600L,
-                                                            16L),
-                                                    Row.ofKind(
-                                                            RowKind.UPDATE_AFTER,
-                                                            "key1",
-                                                            6L,
-                                                            600L,
                                                             19L))
                                             .build())
+                            .runSql(
+                                    "INSERT INTO sink_t SELECT key, val, ts, SUM(val) OVER ("
+                                            + "PARTITION BY key "
+                                            + "ORDER BY key "
+                                            + "ROWS BETWEEN UNBOUNDED PRECEDING AND CURRENT ROW) "
```

Comment:
I think it also makes sense to add this test with a different sort key type for the RANGE operator